sf-docker/sf-observability/configuration/alloy/config.alloy

```diff
@@ -0,0 +1,358 @@
+logging {
+  level  = "info"
+  format = "logfmt"
+}
+
+discovery.docker "flog_scrape" {
+	host             = "unix:///var/run/docker.sock"
+	refresh_interval = "5s"
+}
+
+loki.process "flog_scrape" {
+	forward_to = [loki.write.default.receiver]
+
+	stage.docker { }
+
+	stage.json {
+		expressions = {
+			jsonParsable  = "\"@t\"",
+			levelNonInfo  = "\"@l\"",
+			serviceName   = "ServiceName",
+			sourceContext = "SourceContext",
+		}
+	}
+
+	stage.template {
+		source   = "level"
+		template = "{{ or .levelNonInfo \"Information\" }}"
+	}
+
+	stage.template {
+		source   = "structured"
+		template = "{{ if .jsonParsable }}true{{ else }}false{{ end }}"
+	}
+
+	stage.labels {
+		values = {
+			level         = null,
+			serviceName   = null,
+			sourceContext = null,
+			structured    = null,
+			tenantId      = null,
+		}
+	}
+}
+
+discovery.relabel "flog_scrape" {
+	targets = []
+
+	rule {
+		source_labels = ["__meta_docker_container_label_application"]
+		target_label  = "application"
+		replacement   = "${1}"
+	}
+
+	rule {
+		source_labels = ["__meta_docker_container_name"]
+		regex         = "/(.*)"
+		target_label  = "container"
+	}
+
+	rule {
+		source_labels = ["__meta_docker_container_log_stream"]
+		target_label  = "logstream"
+	}
+
+	rule {
+		source_labels = ["__meta_docker_container_label_monitoringGroup"]
+		target_label  = "monitoringGroup"
+	}
+
+	rule {
+		source_labels = ["__meta_docker_container_label_com_docker_compose_project"]
+		target_label  = "composeproject"
+	}
+
+	rule {
+		source_labels = ["__meta_docker_container_label_com_docker_compose_service"]
+		target_label  = "composeservice"
+	}
+
+	rule {
+		source_labels = ["composeproject"]
+		regex         = "^docker$"
+		target_label  = "composeproject"
+		replacement   = "sf_docker"
+	}
+
+	rule {
+		source_labels = ["composeproject"]
+		regex         = "^sf-docker$"
+		target_label  = "composeproject"
+		replacement   = "sf_docker"
+	}
+
+	rule {
+		source_labels = ["composeproject"]
+		regex         = "^all-in-one$"
+		target_label  = "composeproject"
+		replacement   = "sf_docker"
+	}
+
+	rule {
+		source_labels = ["composeproject"]
+		regex         = "^LFIS$"
+		target_label  = "composeproject"
+		replacement   = "sf_docker"
+	}
+
+	rule {
+		source_labels = ["composeproject"]
+		regex         = "^multi-camera$"
+		target_label  = "composeproject"
+		replacement   = "sf_docker"
+	}
+
+	rule {
+		source_labels = ["composeproject"]
+		regex         = "^multi-server$"
+		target_label  = "composeproject"
+		replacement   = "sf_docker"
+	}
+
+	rule {
+		source_labels = ["composeproject"]
+		regex         = "^rapid-video-processing$"
+		target_label  = "composeproject"
+		replacement   = "sf_docker"
+	}
+
+	rule {
+		source_labels = ["composeproject"]
+		regex         = "^single-camera$"
+		target_label  = "composeproject"
+		replacement   = "sf_docker"
+	}
```

Comment:
I think you can remove these rules, the "application" label is better than the "composeproject", we should throw that away.

Reply:
yes, but it requires also changes made in dashboard, 
and we were having discussion that I should leave it for now :) that it is working and it is not perfect

Reply:
of coruse I can do it, we just agree that I should not for now
should I do it ?

Reply:
I'm sorry, I forgot about our yesterdays conversation.
But looking at it more soberly, I think we should remove it anyway.
The SFIT generated dashboards should still work and the other ones will be easy to fix.